Ai Pin/Stores/ColorStore.swift

```diff
@@ -0,0 +1,27 @@
+import SwiftUI
+
+class ColorStore: ObservableObject {
+    @Published var accentColor: Color = .blue;
+    
+    init() {
+        if let colorHex = UserDefaults.standard.object(forKey: "accentColor") {
```

Comment:
don't use "magic strings" like this. There should be a Constants file somewhere which contains the key. Further, it is advised to prefix the key so it isn't so ambiguous and also to version it. "UI_CUSTOM_ACCENT_COLOR_V1".

Reply:
Noted! Added an app constants file to manage this!